hack/build-images.sh

```diff
@@ -7,7 +7,7 @@ set -o pipefail
 STARTTIME=$(date +%s)
 source_root=$(dirname "${0}")/..
 
-prefix="openshift/origin-"
+prefix="docker.io/openshift/origin-"
```

Comment:
@smarterclayton do i read this https://github.com/openshift/origin/blob/master/hack/build-images.sh#L108 correctly in that origin DOES NOT prefix with 'docker.io'.  What correction do you desire when you say '...This probably belongs in release...' Are you suggesting we should tag prior to releasing with this prefix?